### PR TITLE
Update syntax for ssh command

### PR DESCRIPTION
**Pull request by slemeur**
Updates for the ssh command syntax change.

### CODE REVIEW

File: src/main/_docs/use-che-as-an-ide/ide-ssh.md [hunk 1]
```diff
@@ -46,15 +46,15 @@ One nice aspect of our SSH capabilities is that they are all done inside of a Do
...
-    workspace             # Workspace name or id.  
-    machine-name          # Connect to a secondary machine in the workspace(docker compose)
+    workspace                   # Workspace name or id.  
+    machine-name                # Connect to a secondary machine in the workspace(docker compose)
 # Parameters
-    --url <url>           # {{ site.product_mini_name }} host where workspaces are running
-{% if site.product_mini_cli=="codenvy" %}    --user <login email>        # Codenvy user name
-    --password <login password> # Codenvy password{% endif %}
+    --url <url>                 # {{ site.product_mini_name }} host where workspaces are running
+    --user <login email>        # {{ site.product_mini_name }} user name
```
Comment (JamesDrummond) [inline src/main/_docs/use-che-as-an-ide/ide-ssh.md]: Che does not use login credentials. Thing you need to change line 55-57 back.

Comment (slemeur) [inline src/main/_docs/use-che-as-an-ide/ide-ssh.md]: AFAIK we keep ability to provide credentials in the case Che is used securely.
@benoitf Do you confirm?

Comment (benoitf) [inline src/main/_docs/use-che-as-an-ide/ide-ssh.md]: We might use also a remote secured che (like codenvy with che cli)

Comment (TylerJewell) [inline src/main/_docs/use-che-as-an-ide/ide-ssh.md]: We should always include the full syntax - even secured credentials for the utilities.

**Pull request merged**